tests/foreman/ui/test_repository.py

```diff
@@ -519,33 +521,54 @@ def test_positive_sync_custom_repo_docker(self):
                     self.assertTrue(self.prd_sync_is_ok(repo_name))
 
     @run_only_on('sat')
-    @stubbed()
     @tier1
     def test_positive_create_custom_ostree_repo(self):
         """Create Custom ostree repository.
 
         @id: 852cccdc-7289-4d2f-b23a-7caad2dfa195
 
         @Assert: Create custom ostree repository should be successful
-
-        @caseautomation: notautomated
         """
+        prod = entities.Product(organization=self.organization).create()
+        with Session(self.browser) as session:
+            for repo_name in generate_strings_list():
+                with self.subTest(repo_name):
+                    set_context(session, org=self.organization.name)
+                    self.products.search(prod.name).click()
+                    make_repository(
+                        session,
+                        name=repo_name,
+                        repo_type=REPO_TYPE['ostree'],
+                        url=FEDORA23_OSTREE_REPO,
+                    )
+                    self.assertIsNotNone(self.repository.search(repo_name))
 
     @run_only_on('sat')
-    @stubbed()
     @tier1
     def test_positive_delete_custom_ostree_repo(self):
         """Delete custom ostree repository.
 
         @id: 87dcb236-4eb4-4897-9c2a-be1d0f4bc3e7
 
         @Assert: Delete custom ostree repository should be successful
-
-        @caseautomation: notautomated
         """
+        prod = entities.Product(organization=self.organization).create()
+        repo_name = gen_string('alphanumeric')
+        # Creates new ostree repository using api
+        entities.Repository(
+            name=repo_name,
+            content_type='ostree',
+            url=FEDORA22_OSTREE_REPO,
+            product=prod,
+            unprotected=False,
+        ).create()
+        with Session(self.browser) as session:
+            session.nav.go_to_select_org(self.organization.name)
+            self.products.search(prod.name).click()
+            self.assertIsNotNone(self.repository.search(repo_name))
+            self.repository.delete(repo_name)
 
     @run_only_on('sat')
-    @stubbed()
     @tier1
     def test_positive_update_custom_ostree_repo(self):
```

Comment:
I would divide that case on two as these functionalities are really different. I meant name and URL